src/theme/light-theme.tsx

```diff
@@ -664,45 +668,53 @@ const lightTheme = createTheme({
       styleOverrides: {
         root: {
           '&.Mui-selected': {
-            backgroundColor: alpha(
-              sb_dark_blue,
-              palette.action.selectedOpacity,
-            ),
+            backgroundColor: 'transparent',
+            color: palette.primary.main,
+            fontWeight: font_weight_bold,
             '&:hover': {
-              backgroundColor: alpha(sb_dark_blue, palette.action.hoverOpacity),
+              backgroundColor: sb_secondary_50,
             },
           },
+          '&:hover': {
+            backgroundColor: sb_secondary_50,
+          },
         },
       },
     },
     MuiListItem: {
       styleOverrides: {
         root: {
           '&.Mui-selected': {
-            backgroundColor: alpha(
-              sb_dark_blue,
-              palette.action.selectedOpacity,
-            ),
+            backgroundColor: sb_secondary_50,
             '&:hover': {
-              backgroundColor: alpha(sb_dark_blue, palette.action.hoverOpacity),
+              backgroundColor: sb_base_100,
             },
           },
+          '&:hover': {
+            backgroundColor: sb_secondary_50,
```

Comment:
For these properties, we should use `({ theme }) =>` to use the design token from the palette. At the time that I wrote this existing line, I did not know this was possible.